x-pack/plugins/security/server/routes/authentication/oidc.ts

```diff
@@ -27,58 +27,32 @@ export function defineOIDCRoutes({
   basePath,
   docLinks,
 }: RouteDefinitionParams) {
-  // Generate two identical routes with new and deprecated URL and issue a warning if route with deprecated URL is ever used.
-  for (const path of ['/api/security/oidc/implicit', '/api/security/v1/oidc/implicit']) {
-    const isDeprecated = path === '/api/security/v1/oidc/implicit';
-    /**
-     * The route should be configured as a redirect URI in OP when OpenID Connect implicit flow
-     * is used, so that we can extract authentication response from URL fragment and send it to
-     * the `/api/security/oidc/callback` route.
-     */
-    httpResources.register(
-      {
-        path,
-        validate: false,
-        options: {
-          authRequired: false,
-          excludeFromOAS: true,
-          ...(isDeprecated && {
-            deprecated: {
-              documentationUrl: docLinks.links.security.deprecatedV1Endpoints,
-              severity: 'warning',
-              message: i18n.translate('xpack.security.deprecations.oidcImplicitRouteMessage', {
-                defaultMessage:
-                  'The "{path}" URL is deprecated and will be removed in the next major version. Use "/api/security/oidc/implicit" instead.',
-                values: { path },
-              }),
-              reason: {
-                type: 'migrate',
-                newApiMethod: 'GET',
-                newApiPath: '/api/security/oidc/implicit',
-              },
-            },
-          }),
-        },
+  /**
+   * The route should be configured as a redirect URI in OP when OpenID Connect implicit flow
+   * is used, so that we can extract authentication response from URL fragment and send it to
+   * the `/api/security/oidc/callback` route.
+   */
+  httpResources.register(
+    {
+      path: '/api/security/oidc/implicit',
+      validate: false,
+      options: {
+        authRequired: false,
```

Comment:
__question:__ while you're here, would you mind replacing deprecated `authRequired` with `security.authc.enabled` in all routes in this file?